rpa-builder/6.7/modules/ROOT/pages/toolbox-excel-operations-execute-excel-macro.adoc

```diff
@@ -0,0 +1,62 @@
+= Execute Excel Macro 
+
+Run a macro in an Excel file or sheet. You can specify and pass up to 30 variables to the macro.
+
+== Properties
+
+* *Common* 
++
+See xref:common-properties.adoc[Common Properties] for a description.  
+
+* *Excel Operation Settings*
```

Comment:
This whole setting can be removed (line 11-15), will also be removed in the action step. Was placed there by error from inheriting from the base class while developing.